src/index.php

```diff
@@ -17,8 +16,8 @@
 require_once __DIR__ . '/load.php';
 $di = include __DIR__ . '/di.php';
 
-$url = $di['request']->getQuery('_url') ?? '';
-$http_err_code = $di['request']->getQuery('_errcode') ?? null;
+$url = $_GET['_url'] ?? null;
```

Comment:
This also used to default to an empty string

Reply:
It did - not sure if it matters here - but updated.